services/tests/integration/registry/update-resources.spec.ts

```diff
@@ -23,6 +23,10 @@ jest.mock('child_process', () => ({
   exec: jest.fn((_, cb) => cb()),
 }));
 
+jest.mock('../../../src/modules/directives/policy/opa.ts', () => ({
+  createLoadedPolicy: () => ({ evaluate: () => [{ result: { allow: true } }] }),
```

Comment:
I assume this mock was from a work-in-progress (before latest rebase) that is no longer necessary (there is no `createLoadedPolicy` function there now)